src/client/pages/OfferNew/Checkout/Checkout.stories.tsx

```diff
@@ -24,14 +24,14 @@ export default {
 
 type StoryProps = {
   initialEntries: LocationDescriptor
-  locale: Locale
+  translationsLocale: TranslationsLocale
```

Comment:
IMO we could keep `locale` here :)

Reply:
I think we have too many things names `locale` and it might be confusing?

Reply:
Also, it's clearer that this is the locale for the translations and not a `locale` in the sense it's now used in a bunch of other files?